### ISSUE TITLE
Vagrant test script's ASCII art header renders incorrectly

### ISSUE THREAD

**Issue by codebrewer** [davestephens/ansible-nas#172]
Not exactly critical! The launch script provided to spin up a testing VM prints some ASCII art but this doesn't render correctly, appearing as:

```
                     ._____.   .__
_____    ____   _____|__\_ |__ |  |   ____             ____ _____    ______
\__  \  /    \ /  ___/  || __ \|  | _/ __ \   ______  /    \__  \  /  ___/
 / __ \|   |  \___ \|  || \_\ \  |_\  ___/  /_____/ |   |  \/ __ \_\___ (____  /___|  /____  >__||___  /____/\___  >         |___|  (____  /____  >
     \/     \/     \/        \/          \/               \/     \/     \/
   a n s i b l e - n a s      https://github.com/davestephens/ansible-nas
```

instead of:

```
                     ._____.   .__
_____    ____   _____|__\_ |__ |  |   ____             ____ _____    ______
\__  \  /    \ /  ___/  || __ \|  | _/ __ \   ______  /    \\__  \  /  ___/
 / __ \|   |  \\___ \|  || \_\ \  |_\  ___/  /_____/ |   |  \/ __ \_\___ \
(____  /___|  /____  >__||___  /____/\___  >         |___|  (____  /____  >
     \/     \/     \/        \/          \/               \/     \/     \/
   a n s i b l e - n a s      https://github.com/davestephens/ansible-nas
```

**Comment by codebrewer** [comment]
Addressed by #173

**Comment by davestephens** [comment]
Fixed by #173 👍 (thanks!)
